app.js

```diff
@@ -1,16 +1,25 @@
-const express = require('express')
-const app = express()
-const {getTopics} = require('./controllers/topics-controller')
-const description = require('./endpoints.json')
-
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics-controller");
+const description = require("./endpoints.json");
+const { getArticle } = require("./controllers/articles-controller");
 
 app.use(express.json());
 
-app.get('/api/topics', getTopics)
+app.get("/api/topics", getTopics);
 
-app.get('/api', (req, res, next) => {
-    res.status(200).send({description})
-})
+app.get("/api", (req, res, next) => {
+  res.status(200).send({ description });
+});
 
+app.get("/api/articles/:article_id", getArticle);
+
+app.use((err, req, res, next) => {
+    if (err.status && err.message) {
+        res.status(err.status).send({message: err.message})
+    } else {
+        res.status(400).send({message: 'Bad request'})
```

Comment:
Though this is working now - it may not work down the line and you won't be able to make your messages specific to the type of error. We're getting here in response to a PSQL error, so it's best to check the error code of it and send a message depending on what code it is - have a look in the error handling notes for an example
https://notes.northcoders.com/courses/js-back-end/error-handling#4-refactoring-to-individual-error-handling-blocks